packages/@lwc/template-compiler/src/__tests__/parser.spec.ts

```diff
@@ -465,6 +465,34 @@ describe('expression', () => {
         });
     });
 
+    it('quoted expression with leading spaces should produce error', () => {
+        const { warnings } = parseTemplate(`<template><input title="  {myValue}" /></template>`);
+        expect(warnings[0].message).toMatch(`Ambiguous attribute value title="  {myValue}"`);
+        expect(warnings[0]).toMatchObject({
+            location: EXPECTED_LOCATION,
+        });
+    });
+    // if the curly brace is at 0th or 1st position of the string, then it is currenlty an error (given closing brace exists)
+    // "{username}-lightningUpload-{recId}-"
+    // "[{asdf}]"
```

Comment:
This is ambiguous because it's not clear when "currently" is. The test name seems to be descriptive enough so maybe we can remove this comment?

Reply:
I don't understand this

Reply:
If I read this comment 3 months from now, will it still be true? Or is it only true for the duration of this PR?